### ISSUE TITLE
feedback?

### ISSUE THREAD

**Issue by artofmus** [zauner/vvvv.js#24]
Still need to think about how to implement delay line and in general feedback.

**Comment by geloescht** [comment]
This is already possible with the Delay (HTML5 Audio) node. However it does not make a lot of sense as long as we do not have a mixer node.

**Comment by artofmus** [comment]
Or that I do not understand, or until it is still not possible)

**Comment by artofmus** [comment]
this question, apparently due to the fact that it is impossible within the concept of VVVV make audio feedback node without audio(framedelay) node. This applies to the Web Gl nodes.

**Comment by geloescht** [comment]
Yes, it is possible. Check out the Delay (HTML5 Audio) node. It is possible to create cycles with it. WebGL feedback is an entirely different beast and much harder to implement.

**Comment by artofmus** [comment]
I have it does not work ... Could you @geloescht  send me an example of a patch as it works for you? arturmihel@yandex.ru

**Comment by geloescht** [comment]
Sorry, apparently I was wrong. I tested this with an early version of my implementation and it worked. Matthias removed the delays_output flag when merging my branch. I tried re-adding it, but now I cannot get it to work again. I really have to rework this connection code.
